azure-pipelines.yml

```diff
@@ -0,0 +1,114 @@
+jobs:
```

Comment:
Seems like the strategy and steps for 3 OSes all the same, could they be shared?

Reply:
You could refer to https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=vsts&tabs=yaml

Reply:
updated